src/praisonai/praisonai/profiler.py

```
@@ -1016,6 +1030,33 @@ async def wrapper(*args, **kwargs):
+    if original is None:  # Defensive: the dispatcher is never installed alone.
+        raise RuntimeError("Import profiler dispatcher has no original import hook")
+
+    start = time.time()
+    try:
+        return original(name, globals, locals, fromlist, level)
+    finally:
+        duration_ms = (time.time() - start) * 1000
+        if duration_ms > 1:
+            records = [ImportRecord(module=name, duration_ms=duration_ms) for _ in profilers]
+            with _IMPORT_HOOK_LOCK:
+                for profiler, record in zip(profilers, records):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)